clients.asciidoc

```diff
@@ -562,3 +562,7 @@ https://MyCrypto.com
 Mist is the first ever Ethereum enabled browser, built by the Ethereum Foundation. It also contains a browser-based wallet that was the first ever implementation of the ERC20 token standard (Fabian Vogelsteller, author of ERC20 was also the main developer in Mist). Mist was also the first wallet to introduce the camelCase checksum (EIP-155, see <<eip-155>>). Mist runs a full node, and offers a full DApp browser with support for Swarm based storage and ENS addresses. Find it at:
 
 https://github.com/ethereum/mist
+
+[bibliography]
+=== References
+- [[[1]]] EIP-161: http://eips.ethereum.org/EIPS/eip-161
```

Comment:
Sorry, Are you trying to use an inline anchor?

_Defining an inline anchor_
`[[bookmark-a]]Inline anchors make arbitrary content referenceable.`

Reply:
Yes, an inline anchor to "[1]".

Reply:
Go it! Thanks!